testing/opae-c/test_umsg_c.cpp

```diff
@@ -132,7 +132,7 @@ class umsg_c_p : public ::testing::TestWithParam<std::string> {
     filter_ = nullptr;
     ASSERT_EQ(fpgaInitialize(NULL), FPGA_OK);
     ASSERT_EQ(fpgaGetProperties(nullptr, &filter_), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetObjectType(filter_, FPGA_DEVICE), FPGA_OK);
```

Comment:
Should we have a negative test for this?
Say, when you call this with `FPGA_DEVICE`, it fails?

Reply:
Not a bad idea, but I think that such a test will pass with the current mock.